mlir/include/mlir/Dialect/Affine/Analysis/Utils.h

```diff
@@ -398,7 +403,8 @@ bool buildSliceTripCountMap(
 SliceComputationResult
 computeSliceUnion(ArrayRef<Operation *> opsA, ArrayRef<Operation *> opsB,
                   unsigned loopDepth, unsigned numCommonLoops,
-                  bool isBackwardSlice, ComputationSliceState *sliceUnion);
+                  bool isBackwardSlice, ComputationSliceState *sliceUnion,
+                  FusionStrategy fusionStrategy);
```

Comment:
This isn't proper to do. `computeSliceUnion` shouldn't have anything to do with the fusion strategy or anything fusion related. You'll have to rethink what option to pass if the behavior has to change here.